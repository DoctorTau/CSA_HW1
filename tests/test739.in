478
-9020 -672 -5003 -3204 6479 -9778 -1902 -2086 -9528 3863 8293 -1023 3116 -1817 -3846 -7741 -3903 7189 -7426 -9353 1946 5804 7990 -4668 -7004 9467 1883 8892 -6075 -5949 -9732 5543 -1470 -806 4794 4445 -7017 -4174 1777 -3286 5769 6625 6637 -4054 9909 -833 -7474 2193 -4366 -8774 -1919 3874 -6280 -7357 3596 4315 4583 1173 2051 8631 9823 8312 -5294 -6403 9557 1280 -1971 -6356 -630 -4606 2631 7528 755 2937 -1099 8290 -4703 -5329 8827 8584 9226 -7248 7366 5468 7645 -4245 433 1357 5024 8419 2396 -5415 8948 -427 -5812 4018 4002 -3945 8926 -8953 -5263 7094 -123 -1603 2965 -4244 -8417 -6853 1395 5059 1950 -9390 -5928 -6728 1075 -2407 3852 -3712 -6808 2278 -853 -975 -6220 -9753 9917 8126 -8 4776 -4044 -1188 -8257 4850 6999 300 -8968 -3484 -1670 1074 1168 3975 1037 2485 -7740 7975 9091 5480 3436 -8124 -9968 5338 -5705 1392 -1217 9259 -7542 -8543 -5223 415 4510 749 2091 5066 8239 4490 1298 3436 -3282 8455 -8630 6959 2720 6622 -8106 -3274 -1121 8520 -4969 -2511 2850 5216 7408 -5049 2971 -1202 4152 -4751 -6838 -5739 -9363 -2949 9165 6924 7007 -9808 -5442 -6297 -3619 -2015 9847 1590 9077 6033 6122 4713 356 8162 7814 1249 4676 -9938 8468 8642 -2788 -6628 -3072 -4744 6039 -8567 -1178 8594 -5294 -9682 -2372 -5152 1392 -4807 1516 4652 8090 -5149 7387 2887 8775 -5145 -9063 -165 8685 -1484 -6285 -3052 4582 -6472 115 -1786 9071 -8846 9938 -5077 -6763 -2693 -5408 1494 2655 -6379 2602 -3018 -9425 431 -5503 1020 2107 -682 657 -6783 3059 -7617 -6473 -842 7265 4771 -7828 -8063 -5860 -6398 -8487 -884 -3886 -9876 -3778 8480 1923 228 -8822 1657 7128 -6531 3541 7711 -3937 9202 -5876 6220 -3559 2276 -8362 324 1216 -5913 -3353 -257 -4680 -7587 2546 8242 1361 443 2590 -7195 4124 -1774 6283 1737 7985 -3110 3518 -5003 -4785 -6471 -8585 2590 9254 -8320 9645 9628 -7809 1043 1144 -5590 3742 5641 8686 -1566 2224 4416 -9347 -7593 7563 8636 133 -3826 7427 151 211 4613 7909 4714 6755 -2141 -7877 7874 6512 6792 -3659 6245 214 7977 -3706 6380 201 1549 4038 -319 -8306 -2202 1269 -3603 6984 -2521 -2244 -5868 203 4947 2500 -3621 4468 -1337 2890 3121 -8993 3105 -8806 -6887 8813 5922 8487 4470 -8527 5947 -1118 937 -4225 6244 5491 8341 2096 -2996 -5597 3801 -1512 -17 4023 -7690 713 -793 5047 -5675 8831 -3895 2526 8195 1118 5249 -9192 5747 8237 8412 8861 4470 -7887 4367 7074 -2896 635 4911 6329 9184 5200 9139 5183 8978 -5493 -410 7479 -2634 5922 -8785 4988 9957 401 -1277 -4828 -282 -3617 4936 1054 4428 3867 1093 -57 7712 -9116 -3582 9211 -4866 7304 -895 3213 8855 87 -3470 559 -8518 -8169 6579 -9692 1351 1565 9127 -3056 -2263 3466 971 -5188 8958 4398 -8251 -9185 8744 