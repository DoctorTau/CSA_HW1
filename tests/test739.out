-9020 -672 -3204 -9778 -1902 -2086 -9528 -3846 -7426 -4668 -7004 -9732 -1470 -806 -4174 -3286 -4054 -7474 -4366 -8774 -6280 -5294 -6356 -630 -4606 -7248 -5812 -4244 -9390 -5928 -6728 -3712 -6808 -6220 -8 -4044 -1188 -8968 -3484 -1670 -7740 -8124 -9968 -7542 -3282 -8630 -8106 -3274 -1202 -6838 -9808 -5442 -9938 -2788 -6628 -3072 -4744 -1178 -5294 -9682 -2372 -5152 -1484 -3052 -6472 -1786 -8846 -5408 -3018 -682 -842 -7828 -5860 -6398 -884 -3886 -9876 -3778 -8822 -5876 -8362 -4680 -1774 -3110 -8320 -5590 -1566 -3826 -3706 -8306 -2202 -2244 -5868 -8806 -1118 -2996 -1512 -7690 -9192 -2896 -410 -2634 -4828 -282 -9116 -3582 -4866 -3470 -8518 -9692 -3056 -5188 